src/genie/libs/parser/iosxe/show_lisp.py

```diff
@@ -1582,11 +1585,15 @@ def cli(self, service, instance_id, output=None):
 
         #   Locator  Uptime    State      Pri/Wgt     Encap-IID
         #   10.1.8.8 00:04:02  up          50/50        -
+        # State can be: up, down, route-rejec plus self
         p5 = re.compile(r'(?P<locator>(\S+)) +(?P<uptime>(\S+))'
-                         ' +(?P<state>(up|down))'
+                         ' +(?P<state>(up|down|route-rejec)) *,? *(?P<self>(self)?)'
                          ' +(?P<priority>(\d+))\/(?P<weight>(\d+))'
                          '(?: +(?P<encap_iid>(\S+)))?$')
 
+        # Encapsulating to proxy <device>
+        p6 = re.compile(r'Encapsulating +to +proxy +(?P<device>(\S+)).*$')
```

Comment:
```suggestion
        p6 = re.compile(r'Encapsulating\s+to\s+proxy\s+ETR(\s+Encap-IID\s+(?P<encap_to_petr_iid>\S+))?$')
```
"Encapsulating to proxy ETR" is fixed string and taking 'encap_to_petr_iid' as optional.